codeworld-api/src/CodeWorld/Driver.hs

```diff
@@ -243,6 +243,8 @@ pictureToDrawing (Scale _ x y p) =
     Transformation (scaleDS x y) $ pictureToDrawing p
 pictureToDrawing (Rotate _ r p) =
     Transformation (rotateDS r) $ pictureToDrawing p
+pictureToDrawing (Dilated _ k p) =
```

Comment:
If you pull from github (`git pull --rebase`), then this should already be added.

Reply:
thanks!